spec/models/option_node_spec.rb

```diff
@@ -1,289 +1,298 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe OptionNode do
   include OptionNodeSupport
 
-  describe 'destroy' do
+  describe "destroy" do
     before do
       @node = create(:option_node_with_grandchildren)
       @option = @node.children[0].option
       @node.children[0].destroy
     end
 
-    it 'should not destroy option' do
+    it "should not destroy option" do
       expect(Option.exists?(@option.id)).to be_truthy
     end
   end
 
-  describe 'option_level' do
+  describe "shortcode" do
+    let(:option_set) { create(:option_set, super_multilevel: true) }
+
+    it "should return shortcodes based on sequence" do
+      shortcodes = option_set.descendants.map(&:shortcode)
+      expect(shortcodes).to eq ["1", "2", "3", "4", "5", "6", "7", "8", "9", "a", "b", "c", "d", "e"]
+    end
```

Comment:
Might be good to have a spec demostrating generating shortcodes for large numbers. Perhaps a quick model spec for Base36 directly. (So perhaps making Base36 a model?)